homework_3/src/main/java/com/epam/spring/service/repository/impl/CategoryRepositoryImpl.java

```diff
@@ -0,0 +1,62 @@
+package com.epam.spring.service.repository.impl;
+
+import com.epam.spring.exception.CategoryNotFoundException;
+import com.epam.spring.exception.ReceiptNotFoundException;
+import com.epam.spring.service.model.Category;
+import com.epam.spring.service.repository.CategoryRepository;
+import org.springframework.stereotype.Component;
+
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Component
```

Comment:
Hi Viktor. For the **Repository** level we can use more specific stereotype annotation `@Repository`. Thanks